src/toString.js

```diff
@@ -0,0 +1,13 @@
+const toString = intake => {
+  let output = ""
+
+  if (intake === undefined){
+    return output
+    } else if ( intake === null) {
+      return output
+    } else {
+      return output += intake
+    }
+}
```

Comment:
The ternary operator could be used here (but it's just a matter of taste, the above id perfectly acceptable):
```es6
return intake === null || intake === undefined ? '' + intake : ''
```